sdk/synapse/synapse-artifacts/README.md

```diff
@@ -0,0 +1,26 @@
+## Azure ArtifactsClient SDK for JavaScript
+
+This package contains an isomorphic SDK for ArtifactsClient.
+
+### Currently supported environments
+
+- Node.js version 8.x.x or higher
+- Browser JavaScript
+
+### How to Install
+
+```bash
+npm install @azure/synapse-artifacts
+```
+
+### How to use
+
```

Comment:
would be nice to at least show authenticating the client?

Reply:
Added samples